### PR TITLE
New styles, hero, blocks, fontawesome

### PR DESCRIPTION
**Pull request by TomasKostadinov**
Round 2:

- Added more example configurations

- Added a small hero

- Added blocks for linking things

- Added new sidebar styling

- Added fontawesome

### CODE REVIEW

File: elements.html [hunk 1]
```diff
@@ -90,15 +91,18 @@ <h5>Überschrift 5</h5>
     </div>
     <div class="container">
         <h1>Heros</h1>
+        .hero--small
+        .hero--color
```
Comment (DominikRuf) [inline elements.html:95]: Why is this here?
Not necessary

Comment (TomasKostadinov) [reply elements.html:95]: Documentational purposes

If you open that page, you can see directly what classes are needed to archive this view.

File: elements.html [hunk 2]
```diff
@@ -115,6 +119,129 @@ <h1>Test</h1>
         </div>
     </div>
 
+    <div class="container">
+        <h1>
+            Blöcke .blocks
+        </h1>
```
Comment (TomasKostadinov) [inline elements.html:125]: @DominikRuf same here

File: elements.html [hunk 3]
```diff
@@ -115,6 +119,129 @@ <h1>Test</h1>
...
+                    </div>
+                </div>
+            </a>
+        </div>
+    </div>
+
+    <div class="container">
+        <h1>
+            Blöcke .blocks.blocks--even
+        </h1>
```
Comment (TomasKostadinov) [inline elements.html:155]: and here @DominikRuf

**Review by DominikRuf — APPROVED** [review]
Looks good.
